resume-buddy/src/main/java/com/google/sps/data/EmailTemplates.java

```diff
@@ -41,4 +41,19 @@
           + ""
           + "<p>- ResumeBuddy Team"
           + "</body>";
+
+  public static final String REVIEW_REMINDER_SUBJECT_LINE =
+      "ResumeBuddy - Please review the resume";
+
+  public static final String REVIEW_REMINDER_BODY =
+      "<body style=\"font-family:arial\">"
+          + ""
+          + "<img class=\"logo\" src=\"https://resume-buddy-step-2020.appspot.com/images/logo.png\" alt=\"ResumeBuddy Logo\" width=\"50\">"
+          + ""
+          + "<p>Hi, </p>"
+          + ""
+          + "<p>This is a reminder to review matched user's resume! Please log in to your <a href=\"https://resume-buddy-step-2020.appspot.com/\">account</a> to see more details and review the resume. <p>"
+          + ""
```

Comment:
This mostly a nit: Since these are not raw string literals, there are no newlines captured here. Using `+ ""` doesn't really add anything, if that is the intention. All of this HTML is getting crammed into a single line.

Since there is no requirement for the HTML to have formatting, you could just remove the extra `+ ""` and just make sure this is readable here in the Java. Or if you'd like to make the output HTML human readable, you could either add the `\n` to the strings. Or you could switch this out for Java raw string literal using backticks. e.g.

```
`<body style="font-family:arial">
 ...
</body>`
```